torch_xla/csrc/tensor.cpp

```diff
@@ -670,7 +670,7 @@ void XLATensor::ClearShardingSpec() {
   torch::lazy::Value ir_value = CurrentIrValue();
   if (ir_value) {
     if (ir_value.node != nullptr) {
-      dynamic_cast<XlaNode*>(GetIrValue().node.get())->ClearSharding();
```

Comment:
what's this change for?

Reply:
refactoring, ir_value was already defined a few lines above.